### DISCUSSION TITLE
"Command for chat-instruct mode" - make it available as a small textbox on the Chat tab (left column)

### DISCUSSION THREAD

**Discussion by samvanity** [oobabooga/text-generation-webui#5634] [Ideas]
This special command window is very useful and it should be added to the chat tab. Now there are three columns on the chat interface, and there's a big blank area below the "Past Chats" dropdown on the left. We should include this "Command for chat-instruct mode textbox" there to make it easy to edit.

**Comment by oobabooga** [comment]
That's a good idea, thanks. https://github.com/oobabooga/text-generation-webui/commit/238f69accc254cbad126246eaf2607317b822a1d

**Reply by samvanity** [reply]
great! thanks for listening. Is this going to be available soon?
